packages/ui5-middleware-onelogin/src/types.ts

```diff
@@ -7,5 +7,13 @@ export interface Options {
 		useCertificate: boolean;
 		debug?: boolean;
 		query?: unknown;
+		certificateOrigin?: string;
+		certificateCertPath?: string;
+		certificateCert?: Buffer;
+		certificateKeyPath?: string;
+		certificateKey?: Buffer;
+		certificatePfxPath?: string;
+		certificatePfx?: Buffer;
+		certificatePassphrase?: string;
```

Comment:
We chatted about this one so I'll just mention it, depends on the "design" of the middleware whether or not this should be put into its own "parent"-prop i.e. `certificateOptions` or not. Having it in-line with all other config options as simple as here is also fine with me. Maybe Peter/Volker have some opinions here on how the config(s) should be designed for the middleware. :)

Reply:
I started with subConfig Parameter for multiple Certificates, but for a more simple configuration in yaml and env, i start with this, if there is a feature request we can change it, but my use case only need one certificate :)

Reply:
True that. Better to not overthink something when it isn't required/needed for now anyway. Just wanted to keep that in mind and mention it. :)